app/test_main.py

```diff
@@ -1,3 +1,65 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_age",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="should return [0, 0]"
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="should return [0, 0]"
```

Comment:
Please, choose different names for your tests. `id` stands for identifier and should be unique.